src/docker/templates/compose.js

```diff
@@ -137,9 +138,15 @@ exports.executeTemplate = async ({
   util.logger.debug('Compose modified:', composeConfig);
   util.writeStatus(resultStream, {message: 'Compose file modified', data: composeConfig, level: 'verbose'});
 
+  // generate host
+  const host = getHost({serverConfig, name: baseName, config});
```

Comment:
I'm not sure it makes sense to construct a host here since it's not set at all in compose files 🤔

Reply:
I'd say the best way would be to drop it from here completely.
Thoughts?

Reply:
I think it could make sense to keep it, here's a plausible scenario:

- you have a `docker-compose.yml` template that uses the `$EXOFRAME_HOST` variable inside one of the containers
- the exact same template is reusable for any other exoframe projects - no changes required as `EXOFRAME_HOST` gets autogenerated without conflicts

You could potentially achieve the same using a combination of `EXOFRAME_PROJECT`, `EXOFRAME_USER` and `EXOFRAME_DEPLOYMENT` but I think it could be nice to be consistent with the standard Dockerfile deployments.

If you think this is a stretch I can remove the `host`.

Reply:
That's a fair point 🤔 
What I'm worried about is people thinking that `EXOFRAME_HOST` will be the same as host url set to container. It is already confusing enough as it is with compose files, but I don't know. Maybe I really am overthinking this 🤔

Reply:
Env variables still behave differently in standard Docker deployments vs docker-compose deployments - in first case they're passed into the running container, in the second case they're passed onto the `docker-compose.yml` context and if you want them exposed in the container, you need to do it manually. I think it's more of a matter of good documentation here.

I'm happy to make the change if you decide it's better that way. Let me know.

Reply:
Oooh, man! I must've been to tired last time I looked at the code - I've totally missed that point 😅
This changes everything. This is great!